src/SearchInput/storybook/searchList.ts

```diff
@@ -0,0 +1,30 @@
+export const searchList = [
```

Comment:
Nit: Maybe we could make that list more elaborate so we can see how the fuzzy search actually works?

Reply:
good idea, I've implemented fruits, enjoy! 🍓